README.md

```diff
@@ -1,22 +1,46 @@
-[![Ubuntu 22.04](https://github.com/ROCm/omniperf/actions/workflows/ubuntu-jammy.yml/badge.svg)](https://github.com/ROCm/omniperf/actions/workflows/ubuntu-jammy.yml)
-[![RHEL 8](https://github.com/ROCm/omniperf/actions/workflows/rhel-8.yml/badge.svg)](https://github.com/ROCm/omniperf/actions/workflows/rhel-8.yml)
-[![Instinct](https://github.com/ROCm/omniperf/actions/workflows/mi-rhel9.yml/badge.svg)](https://github.com/ROCm/omniperf/actions/workflows/mi-rhel9.yml)
-[![Docs](https://github.com/ROCm/omniperf/actions/workflows/docs.yml/badge.svg)](https://rocm.github.io/omniperf/)
+[![Ubuntu 22.04](https://github.com/ROCm/rocprofiler-compute/actions/workflows/ubuntu-jammy.yml/badge.svg)](https://github.com/ROCm/rocprofiler-compute/actions/workflows/ubuntu-jammy.yml)
+[![RHEL 8](https://github.com/ROCm/rocprofiler-compute/actions/workflows/rhel-8.yml/badge.svg)](https://github.com/ROCm/rocprofiler-compute/actions/workflows/rhel-8.yml)
+[![Instinct](https://github.com/ROCm/rocprofiler-compute/actions/workflows/mi-rhel9.yml/badge.svg)](https://github.com/ROCm/rocprofiler-compute/actions/workflows/mi-rhel9.yml)
+[![Docs](https://github.com/ROCm/rocprofiler-compute/actions/workflows/docs.yml/badge.svg)](https://rocm.github.io/rocprofiler-compute/)
 [![DOI](https://zenodo.org/badge/561919887.svg)](https://zenodo.org/badge/latestdoi/561919887)
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+# ROCm Compute Profiler
 
-# Omniperf
+## General
+
+ROCm Compute Profiler is a system performance profiling tool for machine
+=======
+# rocprof-compute
+
+## General
+
+rocprof-compute is a system performance profiling tool for machine
+>>>>>>> 412643e9 (Rebranding of github README.md and AUTHORS.)
+=======
+# ROCm Compute Profiler
 
 ## General
-Omniperf is a system performance profiling tool for machine
+
+ROCm Compute Profiler is a system performance profiling tool for machine
+>>>>>>> dae91af3 (Rebranding of top level md files.)
 learning/HPC workloads running on AMD MI GPUs. The tool presently
 targets usage on MI100, MI200, and MI300 accelerators.
 
 * For more information on available features, installation steps, and
 workload profiling and analysis, please refer to the online
-[documentation](https://rocm.github.io/omniperf).
+[documentation](https://rocm.docs.amd.com/projects/rocprofiler-compute/en/latest/).
 
-* Omniperf is an AMD open source research project and is not supported
+<<<<<<< HEAD
+<<<<<<< HEAD
+* ROCm Compute Profiler is an AMD open source research project and is not supported
+=======
+* rocprof-compute is an AMD open source research project and is not supported
+>>>>>>> 412643e9 (Rebranding of github README.md and AUTHORS.)
+=======
+* ROCm Compute Profiler is an AMD open source research project and is not supported
```

Comment:
It looks like there are some unresolved merge conflict that were accidentally checked in.